.eslintrc.json

```diff
@@ -48,6 +48,8 @@
       }
     ],
     "@typescript-eslint/no-non-null-assertion": "off",
+    "@typescript-eslint/no-use-before-define": "warn",
```

Comment:
@hzhu Garden's config is `no-use-before-define` results in an "error". This should probably match.

ref https://github.com/zendeskgarden/eslint-config/blob/a847e6d8feb3a3630455afe6d7af642f4d8018f3/rules/variables.js#L36